src/module_1/module_1_meteo_api.py

```diff
@@ -100,7 +95,7 @@ def daily_to_yearly(daily):
     return yearly
 
 
-def plotting(list):
+def plotting(list, city):
     list1 = daily_to_yearly(list)
```

Comment:
Bad naming:
 - It is not a list
 - The variable name tells nothing on the nature/content of it.